src/components/bar.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const colors = {
+    red:    {background: "#EF5350", border: "#B71C1C"},
+    orange: {background: "#FFA726", border: "#E65100"},
+    yellow: {background: "#FFEE58", border: "#F57F17"},
+    green:  {background: "#66BB6A", border: "#1B5E20"},
+    aqua:   {background: "#26C6DA", border: "#006064"},
+    blue:   {background: "#42A5F5", border: "#0D47A1"},
+    violet: {background: "#7E57C2", border: "#311B92"},
+    magenta:{background: "#AB47BC", border: "#4A148C"},
+    pink:   {background: "#EC407A", border: "#880E4F"},
+    gray:   {background: "rgb(235, 235, 235)", border: "rgb(215, 215, 215)"}
+}
+
+const S = {
+    FlexBarRootContainer: styled.div`
```

Comment:
This code needs to be formatted properly using a formatter

Reply:
The indentation is whack

Reply:
a formatter?

Reply:
yeah like `prettier`. Point being, the indentation is whack.

Reply:
I dont understand what you mean, can you explain?
Is FlexBarRootContainer a bad name, do you want me to create a const for each component, or something else? I dont understand what you mean with "prettier" and that the indentation is whack... sorry

Reply:
The indentation is wrong and any decent formatter would immediately fix it. You don't indent children of an object differently.

Reply:
Got it, fixed it in a recent commit